helm/vertical-pod-autoscaler-app/templates/admission-controller/admission-controller-np.yaml

```diff
@@ -2,9 +2,8 @@ apiVersion: networking.k8s.io/v1
 kind: NetworkPolicy
 metadata:
   labels:
-    app.kubernetes.io/component: admission-controller
-    {{- include "vpa.labels" . | nindent 4 }}
-  name: {{ include "vpa.fullname" . }}-admission-controller
+    {{- include "vertical-pod-autoscaler.admissionController.labels" . | nindent 4 }}
```

Comment:
Is there a way to move the network policy upstream?

Reply:
Technically yes I suppose, but the upstream repo is only maintained by a single person and one cannot create PRs. So we would have to create an issue upstream and see with the maintainer if it's possible.

Reply:
I would keep them here but I'd create the issue also :)

Reply:
We can even technically create a PR or add the netpol we use with a networkPolicy.enabled: false default?

Reply:
Creating a PR upstream will be useless as the owner mentioned : 
> As this is a personal project, as I want to keep some consistency between these charts and as I don't have enough time to write down the best practices I use, I'm not accepting any pull requests.

However I can definitely create an issue